Jenkinsfile

```diff
@@ -28,6 +37,17 @@ for (int i = 0; i < platforms.size(); ++i) {
                     }
                 }
 
+                stage('React Build') {
```

Comment:
The problem here is that you're mixing Jenkinsfile conventions.  The agent syntax comes from declarative pipelines while everything else is scripted.  There are 2 paths to go down here: either convert the pipeline to declarative, or use a sidecar agent to launch and test this <https://www.jenkins.io/doc/book/pipeline/docker/#advanced-usage-with-scripted-pipeline>

I'm not sure if there's a node image in the repo, so I think you might be able to just use the alpine image.

Reply:
Yeah I guess we could go with a declarative syntax.

Reply:
I guess the node example was given on the website examples so don't you think it will run ?

Reply:
The examples are for declarative pipelines not scripted like you're using here.  You can click through on the red x below to see what's failing.  It looks like there are still issues with your file